go/engine/login_provisioned_device.go

```diff
@@ -127,7 +127,14 @@ func (e *LoginProvisionedDevice) Run(ctx *Context) error {
 	}
 
 	// login was successful, unlock the device keys
-	return e.unlockDeviceKeys(ctx, me)
+	err = e.unlockDeviceKeys(ctx, me)
+	if err != nil {
+		return err
+	}
+
+	RunBug3964Repairman(e.G())
```

Comment:
Just to be clear, this is going to run again in `sendNotification` when called via `Login` engine.  I see the config short-circuit, fine to run multiple times, just want to point it out.